src/components/PopupAvatarEdit.js

```diff
@@ -0,0 +1,7 @@
+import Popup from "./Popup.js";
+
```

Comment:
`PopupAvatarEdit.js` is not needed. Popup with forms are handled by `PopupWithForm`